cli/bblfsh-sdk-tools/main.go

```diff
@@ -13,7 +13,7 @@ var version string
 var build string
 
 func main() {
-	parser := flags.NewNamedParser("bblfsh-sdk", flags.Default)
+	parser := flags.NewNamedParser("bblfsh-sdk-tools", flags.Default)
```

Comment:
grep `bblfsh-tools` on `etc` and `test` directory. Replace remaining occurrences, run `make` to update bindata and `go install ./cmd/...`; you can then validate by doing `make test` of a driver